modules/metastorage-api/src/main/java/org/apache/ignite/internal/metastorage/MetaStorageManager.java

```diff
@@ -181,4 +181,11 @@ public interface MetaStorageManager extends IgniteComponent {
      * @return Cluster time.
      */
     ClusterTime clusterTime();
+
+    /**
+     * Returns the local meta storage.
+     *
+     * @return Storage.
+     */
+    LocalMetaStorageManager getLocalStorage();
```

Comment:
This seems to go against the Dependency Injection principle. Why can't local metastorage manager be an independent component/service that gets injected (via constructor) to each component that needs it (including the MetaStorageManager)?